### DISCUSSION TITLE
How can I draw a plot by overlapping it?

### DISCUSSION THREAD

**Discussion by YouHyin** [pybamm-team/PyBaMM#2330] [Q&A] [answered]
I want to draw a graph like the picture I posted, but how can I do it like sim.plot?

![lithuim fast charging](https://user-images.githubusercontent.com/113442395/193040179-4cd385e8-effc-4108-a35f-3febbb095770.png)

![simulation](https://user-images.githubusercontent.com/113442395/193040210-33e1c784-823e-497d-bd2b-6eee6313476c.png)

Thank you so much for always helping me. It helps a lot.

**Comment by valentinsulzer** [comment, marked as answer]
You can't do this with `sim.plot`. You have to use `matplotlib` directly, for example
```python3
plt.plot(sim.solution["Time [s]"].data, sim.solution["Terminal voltage [V]"].data)
```

**Comment by YouHyin** [comment]
Thank you very much.
![코드1](https://user-images.githubusercontent.com/113442395/193057581-d6664eee-11dd-4c4d-a748-a2b35f64bc64.png)
![코드2](https://user-images.githubusercontent.com/113442395/193057610-ae1483c3-5c09-4d02-8c51-6c3fe8d077d7.png)

I made the code like this. However, I couldn't draw a graph of overlapping shapes as I wanted. I'd appreciate it if you could tell me how to fix it.

**Reply by valentinsulzer** [reply]
This becomes a matplotlib question not a pybamm question. They have lots of tutorials available https://matplotlib.org/stable/tutorials/index.html. Maybe someone else will read this and have time to help you
